Content.Server/Explosion/Components/ClusterGrenadeComponent.cs

```diff
@@ -43,6 +43,12 @@ public sealed class ClusterGrenadeComponent : Component
         [ViewVariables(VVAccess.ReadWrite)] [DataField("distance")]
         public float ThrowDistance = 50;
 
+        /// <summary>
+        ///     Use physics impulse instead of
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("directImpulse")]
+        public bool UsePhysics = false;
```

Comment:
Property doesn't seem to do anything as they don't get launched anyway. Better off just having them actually throw? Just use a timer on the system to accumulate when to stop making them dynamic bodytypes and make them KC again.